src/NuGet.Clients/VisualStudio.Facade/ProjectHelper.cs

```diff
@@ -6,12 +6,16 @@
 using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio;
 using Microsoft.VisualStudio.ProjectSystem;
-using Microsoft.VisualStudio.ProjectSystem.Designers;
 using Microsoft.VisualStudio.Shell;
 using MsBuildProject = Microsoft.Build.Evaluation.Project;
 using Task = System.Threading.Tasks.Task;
+#if VS14
```

Comment:
Note: this is the only code delta across the VS versions (that we know of so far)--the API comes from a different DLL now, and the types here live in a different namespace.